src/model/MultiValueColumn.ts

```diff
@@ -50,33 +50,36 @@ export class CustomSortCalculation {
   }
 }
 
-// function getPercentile(data, percentile) {
-//
-//       var index = (percentile / 100) * data.length;
-//       var result;
-//       if (Math.floor(index) === index) {
-//         result = (data[(index - 1)] + data[index]) / 2;
-//       } else {
-//         result = data[Math.floor(index)];
-//       }
-//       return result;
-//     }
-
 function numSort(a, b) {
   return a - b;
 }
 
+enum Sort {
+
+  min, max, median, q1, q3
+}
+
+
+interface boxPlotData {
+
+  min: number;
+  max: number;
+  median: number;
+  q1: number;
+  q3: number;
+}
 
 export default class MultiValueColumn extends ValueColumn<number[] > {
   private sortBy;
```

Comment:
if you define fields without an initializer, please define its type, such that the compiler can complain